mep/people/templates/people/member_detail.html

```diff
@@ -0,0 +1,51 @@
+{% extends 'base.html' %}
+
+{% block page-subtitle %}{{ member.name }} · {% endblock %}
```

Comment:
Maybe HTML title should be `name · Library Members · S&co` ?

Reply:
seems long for a title that might go on a tab? afaik we aren't planning on having people's names map to any other kind of page than a member, so 'Library Members' seems redundant to me, but I'm good either way

Reply:
I don't think of labels as for tabs primarily (my tabs are never long enough for that!) but for browser history navigation, search engine listings, and preview links on social sharing. I think the extra context is valuable, and also maps to the hierarchy of the site.

Reply:
ah, makes sense re:hierarchy, especially given what we're doing with the landing pages. I'll change it